codejail/languages.py

```diff
@@ -0,0 +1,123 @@
+"""
+Language-specific configuration for codejails.
+"""
+
+from collections import namedtuple
+import textwrap
+
+
+Language = namedtuple('Language', ['name', 'argv', 'safe_exec_template'])
+
+
+# Allow lowercased names at module level
+# pylint: disable=invalid-name
+
+other = Language(
+    name='other',
+    argv=[],
+    safe_exec_template=None,
+)
+
+python2 = Language(
+    name='python2',
+    argv=[
+        '-E',  # Ignore the environment variables PYTHON*
+        '-B',  # Don't write .pyc files.
+    ],
+    safe_exec_template=textwrap.dedent(
+        """
+        import sys
+        try:
+            import simplejson as json
+        except ImportError:
+            import json
+        """
+        # We need to prevent the sandboxed code from printing to stdout,
```

Comment:
Here's the awkward thing about reviewing this PR: scrunching up my nose at strange code style, and realizing it was that way when *I* wrote it....